playwright-e2e/tests/atcp/atcp-regression-tests.spec.ts

```diff
@@ -0,0 +1,429 @@
+import { test, expect } from '@playwright/test';
+import { setAuth0UserEmailVerified } from 'utils/api-utils';
+import { APP } from 'data/constants';
+import * as auth from 'authentication/auth-base';
+import { generateEmailAlias } from 'utils/faker-utils';
+import * as testutils from 'utils/test-utils';
+
+const { AT_USER_EMAIL, AT_USER_PASSWORD } = process.env;
+
+test('ATFamiles test statics', async ({ page }) => {
+  await page.goto('https://atcp.test.datadonationplatform.org/');
```

Comment:
This is all pretty much a straight dump from playwright generator.  We'll want to break this out into page classes like the other tests, so it's very much a WIP.

Reply:
Might actually want to move it to another PR.